### ISSUE TITLE
Project moved to https://github.com/AlexanderWillner/kingraph

### ISSUE THREAD

**Issue by janisz** [rstacruz/kingraph#18]
It looks like this project is dead but it's continued by @AlexanderWillner
https://github.com/AlexanderWillner/kingraph

**Comment by vorburger** [comment]
https://github.com/elenawinters/kingraph has something more recent; see also discussion in https://github.com/elenawinters/kingraph/pull/2, and my https://github.com/vorburger/kingraph.

**Comment by vorburger** [comment]
@rstacruz perhaps we can use this issue here on your original repo instead of the closed https://github.com/elenawinters/kingraph/pull/2 in @elenawinters, to continue further discussion about what was started there?

Re. _kingraph name on NPM,_ I personally don't have any particular interest in or use for that, at least for now. Full Disclosure: I just want this for personal use, but am very happy to share what I am doing. And not looking for additional project to spend too much time to maintain. 😈 

Re. _happy to make an edit on the readme for my repo to point to the most active fork,_ that fork currently appears to be my own. I had a brief look over [the Network Graph](https://github.com/rstacruz/kingraph/network), and started with @elenawinters `HEAD`, which also includes work by @marekkalnik @raisedadead @vabene1111 @AlexanderWillner @constantinmaes.

I was wondering if @AlexanderWillner re. _"Might make sense to have a GitHub Kingraph Organization - but at the end, doesn't matter which repo to improve step by step."_ might be something you would consider doing? Concretely, create an org, and move this rstacruz/kingraph repo under that org? https://github.com/kingraph seems to be already taken, is that yours, or a squatter? 🤣 

@rstacruz if you are concerned about any additional workload this could generate, rest assured that I'm not "expecting" anything 🍭 of you... it's up to you if you then want to invite an additional contributors to an Org. I would be honoured, if you so chose. It's also totally "fair", at least to me, to then let such additional new contributors decide what they want to merge, what pull requests they would like to review (or not), which issues they want to engage on (or not) - we're all busy, everyone's time is limited. But we're more together than the sum of each of us. I have used a model like this on https://github.com/lastnpe, where I basically pretty quickly invited anyone who sent 2-3 PRs to become an additional commiter, and at least for that community, that worked out well, for me; I launched it, found other people interested - and am currently not involved in it anymore myself! 

So to be even more explicit: If you did create an org and invite me as a co-owner/admin, I personally would probably push my changes there instead of my fork, and review anyone else's PRs as long as they build and don't break existing tests [I've set an Action on my fork], and invite such contributors as new committers. I myself likely would not have the time to offer much "support" on issues. Other additional committers might though. You wouldn't really have to do anything additional, after creating the org and moving the repo - unless new activity motivates you to, obviously.

@momack2 FYI (given your at least past interest on some of the open issues)

**Comment by AlexanderWillner** [comment]
> I was wondering if @AlexanderWillner re. _"Might make sense to have a GitHub Kingraph Organization - but at the end, doesn't matter which repo to improve step by step."_ might be something you would consider doing? Concretely, create an org, and move this rstacruz/kingraph repo under that org? https://github.com/kingraph seems to be already taken, is that yours, or a squatter? 🤣

No. Also no (public) activity since it was registered in 2014...

**Comment by rstacruz** [comment]
Good points - moving to an org definitely makes sense! Given `@kingraph` is taken, `@kingraph-diagram` might be a good alternative. Any thoughts on the name?

**Comment by marekkalnik** [comment]
It may be possible to get kingraph if you want it ?

<img width="725" alt="image" src="https://user-images.githubusercontent.com/548629/193817041-70913d76-7549-4e79-b87e-036fb7c9bbc4.png">

**Comment by rstacruz** [comment]
I'll move the repo to `kingraph-diagram/kingraph`, and publish the npm package as `@kingraph/kingraph`. The `kingraph` npm package name has been taken by another fork.

**Comment by AlexanderWillner** [comment]
Now I have: ```alexander.willner```

**Comment by giganteous** [comment]
any progress on this?

**Comment by AlexanderWillner** [comment]
I think @rstacruz wanted to setup an NPM package ([not done I think](https://www.npmjs.com/search?q=kingraph)) and a organisation [kingraph-diagram](https://github.com/orgs/kingraph-diagram) seems to exist...but no repo yet

**Comment by hasangokdag** [comment]
leaving a comment to see where this goes

**Comment by AlexanderWillner** [comment]
Might make sense to create another organization. Depends a bit whether there are volunteers interested in joining.

**Comment by giganteous** [comment]
Heya,Im not particular interested in contributing: i rewrote most of the code in python as I generally have no nodejs installed. I did find a bug (really an unused code path), if you're interested I could writethat down. Nice to see others picking this up: it is very lean.regardskaiSent from my iPhoneOn 4 Jun 2024, at 14:12, Alex ***@***.***> wrote:﻿
Might make sense to create another organization. Depends a bit whether there are volunteers interested in joining.

**Comment by AlexanderWillner** [comment]
A rewrite in Python sounds appealing...

**Comment by giganteous** [comment]
I've uploaded my python "fork" to https://gitlab.com/giganteous/famstam. It took some effort to get roughly the same output with my large family, but I managed.

I don't consider myself a programmer: there's probably an even easier way to template this using jinja or something.
